packages/components/src/border-box-control/border-box-control/component.tsx

```diff
@@ -147,22 +147,11 @@ const UnconnectedBorderBoxControl = (
 };
 
 /**
- * The `BorderBoxControl` effectively has two view states. The first, a "linked"
- * view, allows configuration of a flat border via a single `BorderControl`.
- * The second, a "split" view, contains a `BorderControl` for each side
- * as well as a visualizer for the currently selected borders. Each view also
- * contains a button to toggle between the two.
- *
- * When switching from the "split" view to "linked", if the individual side
- * borders are not consistent, the "linked" view will display any border
- * properties selections that are consistent while showing a mixed state for
- * those that aren't. For example, if all borders had the same color and style
- * but different widths, then the border dropdown in the "linked" view's
- * `BorderControl` would show that consistent color and style but the "linked"
- * view's width input would show "Mixed" placeholder text.
```

Comment:
Could the behaviour previously described here be unexpected at all? What's the thinking behind removing this additional information and context?

Is the reasoning for removing this just to avoid a wordy description in IDE hints etc? In that case, would it be better to move this information to the readme?

Reply:
If you look at the README (see https://github.com/WordPress/gutenberg/blob/b7f9af87f212b83c68d6f1b92daa6d9ee1e2024d/packages/components/src/border-box-control/border-box-control/README.md), you'll see that there's a general description and then a "Development guidelines" section. This component breaks from the norm by having the "development guidelines" as JSDoc instead of the main description of the component.

My change here is just to bring consistency with what we normally do. Developers will still be able to read those "development guidelines" where they are used to, like with any other components.